spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
ah okay my bad, I misunderstood :confounded:
anyway let's go with ```let(:level) { an_instance_of(Fixnum) }``` :+1: 
it's indeed better than just ```let(:level) { Fixnum }```